test/bucket_test.exs

```diff
@@ -60,24 +62,24 @@ defmodule Bucket.BucketTest do
     assert Bucket.between("1|a", "1|c") == "1|b"
     assert Bucket.between("1|0", "1|z") == "1|U"
     assert Bucket.between("2|AA", "2|AB") == "2|AAV"
-    assert Bucket.between("2|aaaaz", "2|zzzzz") == "2|nIIIT"
+    assert Bucket.between("2|aaaaz", "2|zzzzz") == "2|nIIIU"
```

Comment:
why did this one change?

Reply:
Because there was a bug which leads to incorrect middle value calculation in some cases which I fixed. Lets verify it:
```
LexorankEx.Bucket.distance("2|aaaaz", "2|nIIIT") == 187732124
LexorankEx.Bucket.distance("2|zzzzz", "2|nIIIT") == 187732126
```
As you see distance between lowest and smallest values is not equal, moreover its difference is 2
Now, lets check correct value:
```
LexorankEx.Bucket.distance("2|aaaaz", "2|nIIIU") == 187732125
LexorankEx.Bucket.distance("2|zzzzz", "2|nIIIU") == 187732125
```
Distance is equal, it means that middle value is calculated correctly.

Reply:
oh, I see, thx!